[2023-06-30T06:04:16.476+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: order-letter_NER.post_to_dynamics_task manual__2023-06-30T05:33:07.927789+00:00 [queued]>
[2023-06-30T06:04:16.487+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: order-letter_NER.post_to_dynamics_task manual__2023-06-30T05:33:07.927789+00:00 [queued]>
[2023-06-30T06:04:16.488+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 6
[2023-06-30T06:04:16.522+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): post_to_dynamics_task> on 2023-06-30 05:33:07.927789+00:00
[2023-06-30T06:04:16.535+0000] {standard_task_runner.py:57} INFO - Started process 2468 to run task
[2023-06-30T06:04:16.542+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'order-letter_NER', 'post_to_dynamics_task', 'manual__2023-06-30T05:33:07.927789+00:00', '--job-id', '59', '--raw', '--subdir', 'DAGS_FOLDER/order-letter-extraction.py', '--cfg-path', '/tmp/tmpj2afffim']
[2023-06-30T06:04:16.547+0000] {standard_task_runner.py:85} INFO - Job 59: Subtask post_to_dynamics_task
[2023-06-30T06:04:16.679+0000] {task_command.py:410} INFO - Running <TaskInstance: order-letter_NER.post_to_dynamics_task manual__2023-06-30T05:33:07.927789+00:00 [running]> on host f7b244b51c6e
[2023-06-30T06:04:16.964+0000] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Abin Jilson' AIRFLOW_CTX_DAG_ID='order-letter_NER' AIRFLOW_CTX_TASK_ID='post_to_dynamics_task' AIRFLOW_CTX_EXECUTION_DATE='2023-06-30T05:33:07.927789+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-06-30T05:33:07.927789+00:00'
[2023-06-30T06:04:18.232+0000] {logging_mixin.py:149} INFO - Error: 401 <error xmlns="http://docs.oasis-open.org/odata/ns/metadata"><code>Unauthorized</code><message>The credentials provided are incorrect</message></error>
[2023-06-30T06:04:18.238+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-06-30T06:04:18.257+0000] {taskinstance.py:1345} INFO - Marking task as SUCCESS. dag_id=order-letter_NER, task_id=post_to_dynamics_task, execution_date=20230630T053307, start_date=20230630T060416, end_date=20230630T060418
[2023-06-30T06:04:18.332+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-06-30T06:04:18.356+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
